### PR TITLE
feat: PUT /delay_penalty_change

### PR DESCRIPTION
**Pull request by YumeT023**

### CODE REVIEW

File: src/main/java/school/hei/haapi/service/DelayPenaltyService.java
Comment (haritianaadriano) [inline src/main/java/school/hei/haapi/service/DelayPenaltyService.java:1]: can you make the method getCurrentDelayPenalty() to protected for the scop

**Review by haritianaadriano — CHANGES_REQUESTED** [review]
can you make the method getCurrentDelayPenalty() to protected for the scop

File: src/main/java/school/hei/haapi/service/DelayPenaltyService.java
Comment (YumeT023) [reply src/main/java/school/hei/haapi/service/DelayPenaltyService.java:1]: uhmm... Okay, but why protected instead of public ?

Comment (haritianaadriano) [reply src/main/java/school/hei/haapi/service/DelayPenaltyService.java:1]: i dont' really know but for protect the data ?

Comment (YumeT023) [reply src/main/java/school/hei/haapi/service/DelayPenaltyService.java:1]: `protected` access modifier mean that it remains protected but can be accessed by its child class

**Review by haritianaadriano — APPROVED** [review]

**Pull request merged**